packages/flutter/test/foundation/leak_tracking.dart

```diff
@@ -5,8 +5,58 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:leak_tracker/leak_tracker.dart';
+import 'package:meta/meta.dart';
 
-typedef LeaksObtainer = void Function(Leaks foundLeaks);
+// The test_api package is not for general use... it's literally for our use.
+// ignore: deprecated_member_use
+import 'package:test_api/test_api.dart' as test_package;
```

Comment:
Looks like this is only needed for the `Timeout` type below? I believe that type is exported from `package:flutter_test/flutter_test.dart` above somewhere, so you shouldn't need this...

Reply:
nope. I am getting errors without it.

Reply:
What's the error? That's something we should fix because users will otherwise not be able to use this without importing this deprecated packages either.

Reply:
Undefined class 'test_package.Timeout'.
Try changing the name to the name of an existing class, or creating a class with the name 'test_package.Timeout'.

Reply:
fixed. thanks